src/main/scala/kamon/newrelic/spans/NewRelicSpanConverter.scala

```diff
@@ -19,9 +19,9 @@ import kamon.util.Clock
 object NewRelicSpanConverter {
 
   /**
-   * Converts a Kamon Span$Finished instance into a New Relic Span
+   * Converts a Kamon Span.Finished instance into a New Relic Span
    *
-   * @param kamonSpan Kamon Span$Finished
+   * @param kamonSpan Kamon Span.Finished
```

Comment:
rad

Reply:
sbt was yelping about it, so I fixed er up